src/components/Sum.jsx

```diff
@@ -1 +1,18 @@
+import React from 'react';
 // export function Sum() {}
+
+export function Sum({
+  a = 0,
+  b = 0,
+}) {
```

Comment:
```suggestion
export function Sum({ a = 0, b = 0 }) {
```
You can make it in one line if there are < 3 props